runtime/onert/core/src/compiler/Compiler.cc

```diff
@@ -82,7 +82,13 @@ void verboseOptions(compiler::CompilerOptions &options)
   VERBOSE(Compiler) << "fp16_enable              : " << options.fp16_enable << std::endl
                     << std::noboolalpha;
 }
+} // namespace
 
+namespace onert
+{
+
+namespace compiler
+{
```

Comment:
The diff maybe looks confusing. It just moves `setBackendMap` from anonymous namespace to `compiler` namespace so that it can be used out of this file. (`nnfw_api_internal.cc`).